Description: Do not download virtualenv when testing
 The tests_require of the setup.py upstream is trying to download a virtualenv
 tarball. This patch removes this.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-03-09

--- python-pecan-0.4.5.orig/setup.py
+++ python-pecan-0.4.5/setup.py
@@ -50,7 +50,6 @@ except:
 
 
 tests_require = requirements + [
-    'virtualenv',
     'gunicorn',
     'mock',
     'sqlalchemy'
